.github/workflows/validate_markdown.yaml

```diff
@@ -0,0 +1,36 @@
+name: Markdown validation
+
+on:
+  workflow_call:
+    inputs:
+      SYNTAX_CHECK_IGNORE_FILES:
+        description: 'Multiple files must be separated with single space, eg "./one_file.md ./another_file.md"'
+        required: false
+        type: string
+        default: ''
+
+jobs:
+  markdown_validation:
+    name: Markdown validation
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check out code
+      uses: actions/checkout@v2
+
+    - name: Set up config files
+      run: |
+        wget https://raw.githubusercontent.com/uc-cdis/.github/feat/lint-md/.github/workflows/validate_markdown_lint.jsonc -P .github/workflows/
+        wget https://raw.githubusercontent.com/uc-cdis/.github/feat/lint-md/.github/workflows/validate_markdown_links.json -P .github/workflows/
```

Comment:
pulling these here and running the lint right after in this workflow means that repos don't have the option of overriding or modifyig the config. In this case that's probably fine, but like for Super Lint b/c there's so many linters and probably different repos may want diff standards at first, I was trying to build in some flexibility to override the defaults. All that said, for the markdown linting I'm happy forcing the global config

Reply:
yes i didn't have time to add configuration on friday, and i thought it'd be fine to merge it as-is and add configuration later if needed. plus this way all the repos are consistent

Reply:
Yup, agreed